packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -252,6 +253,42 @@ export class InterfaceVpcEndpointService implements IInterfaceVpcEndpointService
   }
 }
 
+// The following services need to have their interface endpoints prefixed with 'cn' (before com.amazonaws.{REGION}) for china regions
+const CN_OVERRIDES = ['application-autoscaling',
+  'autoscaling',
```

Comment:
This list will be hard to maintain, is this documented anywhere? 
Questions/Considerations:

1. Can we simply add the partition regardless of the service and partition? Using `Aws.PARTITION` .
2. If the partition is required only in the china partition, can we only append it if the stack is created in the china partition?